MapboxCoreNavigation/Constants.swift

```diff
@@ -110,81 +110,15 @@ public var MaxSecondsSpentTravelingAwayFromStartOfRoute: TimeInterval = 3
 
 
 /**
- Distance in meters for the minimum length of a step for giving a `medium` alert while using `MBDirectionsProfileIdentifierAutomobile` or `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic`.
+ Remaing distance on a motorway at which the `.high` `AlertLevel` will be given. This overrides `RouteControllerHighAlertInterval` only when the current step is a motorway. Default value is a half mile.
  */
-public var RouteControllerMinimumDistanceForMediumAlertDriving: CLLocationDistance = 400
+public var RouteControllerMotorwayHighAlertDistance: CLLocationDistance = 804.672
 
 
 /**
- Distance in meters for the minimum length of a step for giving a `medium` alert while using `MBDirectionsProfileIdentifierCycling`.
+ Remaing distance on a motorway at which the `.medium` `AlertLevel` will be given. This overrides `RouteControllerMediumAlertInterval` only when the current step is a motorway. Defauly value is 2 miles.
  */
-public var RouteControllerMinimumDistanceForMediumAlertCycling: CLLocationDistance = 200
```

Comment:
This PR effectively reverts #82. Have we concluded that the driving profile’s thresholds are suitable for cycling and walking too?